needle/engines/perceptualdiff_engine.py

```diff
@@ -22,8 +22,13 @@ def assertSameFiles(self, output_file, baseline_file, threshold):
         process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         perceptualdiff_stdout, _ = process.communicate()
 
-        if process.returncode == 0:
-            # No differences found
+        # Sometimes perceptualdiff returns a false positive with this exact message:
+        # 'FAIL: Images are visibly different\n0 pixels are different\n\n'
+        # We catch that here.
+        if process.returncode == 0 or '\n0 pixels are different' in perceptualdiff_stdout:
```

Comment:
This looks like we have to use `b'\n0 pixels…` – confirming on the test matrix now